packages/transformers/babel/src/babel7.js

```diff
@@ -74,6 +76,28 @@ export default async function babel7(
     },
   };
 
+  if (tracer.enabled) {
+    config.wrapPluginVisitorMethod = (
```

Comment:
Does this wrap visiting each individual AST node, or just once for each plugin? How expensive is that?

Reply:
> Does this wrap visiting each individual AST node, or just once for each plugin? How expensive is that?

I just ran two full production Jira builds back-to-back, one with tracing & one without (both with `--no-cache`). The timings were 18m 29s without tracing, and 19m 1s with tracing (on an M1 Mac). So minimal overhead for tracing in general. 

The trace file produced is approximately 1.4GB - can't load it directly in the Perfetto UI (the WASM version), but works fine via the `trace_processor` CLI tool.

There are approximately 2.9m Babel transform entries - so yes, my understanding is that it wraps every visitor call for every AST type, but as you can see the overhead of this is relatively minimal.